x-pack/plugins/security_solution/public/alerts/pages/detection_engine/rules/details/index.tsx

```diff
@@ -258,6 +258,21 @@ export const RuleDetailsPageComponent: FC<PropsFromRedux> = ({
     return null;
   }
 
+  // TODO: remove this once namespace is finalized as camecased or snakecase
+  const exceptionListsMeta = useMemo(() => {
+    const ruleExceptionListReferences = rule?.exceptions_list;
+    if (ruleExceptionListReferences) {
+      return ruleExceptionListReferences.map((reference) => {
```

Comment:
Super nit suggestion: you can destructure it here so it's `ruleExceptionListReferences.map(({ id, namespace_type }) =>`